### PR TITLE
fix hello-world container

### PR DESCRIPTION
**Pull request by devimc**
Some containers for example 'hello-world' does not
contain a passwd file for this reason hyperstart
should provide a default user information (struct passwd)
to allow the containers start without errors

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

File: src/util.c [hunk 1]
```diff
@@ -27,6 +27,19 @@
 #define STATELESS_PASSWD_FILE "/usr/share/defaults/etc/passwd"
 #define STATELESS_GROUP_FILE "/usr/share/defaults/etc/group"
 
+/*
+ * This struct is returned when passwd file
+ * does not exist for example hello-world container
```
Comment (sboeuf) [inline src/util.c]: I would say
```
// This structure defines the default password.
// It is used when no password file could be found.
```

**Review by jcvenegas — APPROVED** [review]
lgtm

**Comment by amshinde** [comment]
After addressing @sboeuf comment,
lgtm

**Comment by sboeuf** [comment]
@devimc I think the PR does the job, but I am wondering if it would not be better to fix this the other way. I mean to create a password file (regular or stateless, I don't get the difference) for this hello world container ? Is there anything preventing for such a file to be created in case of hello world container ?

**Comment by devimc** [comment]
@sboeuf '/etc' directory does not exist in that container modifying rootfs is not a good idea because user could export that container with the extra directory ('/etc') , from my point of view runtime must not modify the rootfs

**Comment by sboeuf** [comment]
@devimc Ok makes sense, you have added a fallback's fallback, that's fine I guess.
LGTM

**Comment by amshinde** [comment]
@devimc 
Some containers eg hello-world container may not have passwd file.
Hyperstart should not fail when this file does not exist, instead root user
must be used by default to spawn the container.

**Pull request merged**